index.html

```diff
@@ -518,12 +518,12 @@ <h2>Security Considerations</h2>
     information to compute "remaining time" within any callback.</p>
     <p>To mitigate cache and statistical fingerprinting attacks, the resolution
     of the time estimates returned by the <a>IdleDeadline</a> interface should
-    be set to the same 5 microsecond minimum as the <code>Performance</code> interface
-    defined in [[HR-TIME]].</p>
+    be [=coarsen time|coarsened=] to ensure it is protected by the
```

Comment:
maybe "coarsened based on cross-origin isolated capability, to ensure this API is not exposing more information than what's exposed by other web-exposed timers"?

Reply:
Accepted